inference-engine/tests/functional/plugin/shared/src/behavior/cpp_holders.cpp

```diff
@@ -76,8 +80,83 @@ EXPECT_EXIT(_statement; exit(0), testing::ExitedWithCode(0), "")
             release(i);
     }
 
+    void release_order_test_import_network(
+            std::vector<int> order, const std::string &deviceName,
+            std::shared_ptr<ngraph::Function> function) {
+        InferenceEngine::CNNNetwork cnnNet(function);
+        InferenceEngine::Core core;
+        std::string blobFileName { "compiled_blob.blob" };
+        {
+            auto exe_net = core.LoadNetwork(cnnNet, deviceName);
+            std::ofstream file(blobFileName, std::ios::binary);
+            exe_net.Export(file);
+        }
+        auto exe_net = core.ImportNetwork(blobFileName, deviceName);
```

Comment:
It looks like this code should work:

```
    std::ostringstream buf;
    {
        auto exe_net = core.LoadNetwork(cnnNet, deviceName);
        exe_net.Export(buf);
    }

    auto exe_net = core.ImportNetwork(buf.str(), deviceName);
```

To avoid `std::remove(blobFileName.c_str());`. Also file creation seems redundant.

Reply:
good point, updated.